#! /usr/bin/env python

PACKAGE='seabee3_physics'
import roslib; roslib.load_manifest(PACKAGE)

from math import pi

from driver_base.msg import SensorLevels
from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()
#Name            Type   Reconfiguration level             Description         Default Min Max
gen.add( "thrust_to_force",  double_t, SensorLevels.RECONFIGURE_RUNNING, "Conversion from thrust to force", 0.11, 0.0, 1.0 )
gen.add( "drag_x", double_t, SensorLevels.RECONFIGURE_RUNNING, "", 5.0, 0.0, 10.0 )
gen.add( "drag_y", double_t, SensorLevels.RECONFIGURE_RUNNING, "", 5.0, 0.0, 10.0 )
gen.add( "drag_z", double_t, SensorLevels.RECONFIGURE_RUNNING, "", 5.0, 0.0, 10.0 )
gen.add( "linear_damping", double_t, SensorLevels.RECONFIGURE_RUNNING, "", 0.0, 0.0, 1.0 )
gen.add( "angular_damping", double_t, SensorLevels.RECONFIGURE_RUNNING, "", 0.3, 0.0, 1.0 )

exit(gen.generate(PACKAGE, "dynamic_reconfigure_node", "Seabee3Physics"))


